packages/playwright-core/src/server/injected/injectedScript.ts

```diff
@@ -856,6 +856,33 @@ export class InjectedScript {
     return error;
   }
 
+  highlight(selector: ParsedSelector) {
+    this.hideHighlight();
+    (window as any).pw_highlights = [];
+    const elements = this.querySelectorAll(selector, document.documentElement);
+    for (const element of elements) {
+      const box = element.getBoundingClientRect();
+      const div = document.createElement('div');
```

Comment:
Use `x-pw-debug-highlight` instead to avoid accidental styling from the page's css?